media/js/wiki.js

```diff
@@ -302,6 +302,8 @@
         var $pageButtons = $('.page-buttons');
         var pageButtonsOffset = $pageButtons.offset();
 
+        var stickyFeatureEnabled = $pageButtons.attr('data-sticky') === 'true';
```

Comment:
Not ready for public consumption yet because of RTL problems so it's not blocking this PR :)